js/qmwp-bargraph-gauge-scatterplot-timeline_charts.js

```diff
@@ -27,34 +27,34 @@ AnalyzeChart = function () {
     var tlEnableHorizontalGuides;
 
     // Scatterplot settings
-    var spShowLinearRegression
+    var spShowLinearRegression;
 
     var retrieveSettings = function () {
-        if (typeof(Storage) !== "undefined") {
-            tlEnableMarkers = (localStorage["tlEnableMarkers"] || "true") == "true" ? true : false;					// On by default
-            tlSmoothGraph = (localStorage["tlSmoothGraph"] || "true") == "true" ? true : false;						// On by default
-            tlEnableHorizontalGuides = (localStorage["tlEnableHorizontalGuides"] || "false") == "true" ? true : false;			// Off by default
-            spShowLinearRegression = (localStorage["spShowLinearRegression"] || "true") == "true" ? true : false;			// On by default
+        if (typeof(Storage) !== 'undefined') {
+            tlEnableMarkers = (localStorage['tlEnableMarkers'] || 'true') == 'true' ? true : false;					// On by default
+            tlSmoothGraph = (localStorage['tlSmoothGraph'] || 'true') == 'true' ? true : false;						// On by default
+            tlEnableHorizontalGuides = (localStorage['tlEnableHorizontalGuides'] || 'false') == 'true' ? true : false;			// Off by default
+            spShowLinearRegression = (localStorage['spShowLinearRegression'] || 'true') == 'true' ? true : false;			// On by default
```

Comment:
Line is too long.<br>['spShowLinearRegression'] is better written in dot notation.